### PR TITLE
Add a test where RETURNDATACOPY with a nonzero size fails after a CAL…

### PR DESCRIPTION
**Pull request by pirapira**
…L with insufficient balance

### CODE REVIEW

File: src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_following_too_big_transferFiller.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                            "0x00" : "0x01"
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+            "0x0aabbccdd5c57f15886f9b263e2f6d2d6c7b5ec6" : {
+				"balance" : "0x00",
+                "code" : "{ (seq (MSTORE 0 0x0000111122223333444455556666777788889999aaaabbbbccccddddeeeeffff) (RETURN 0 32)) }",
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_following_too_big_transferFiller.json:33]: do ve have a test where return data is checked when the call fails on other reasons? 
like with REVERT? 

or where this subcall has another subcall that sets return data but then the first subcall fails 
and thus all first subcall should be reverted and in this scenario we should check the return data in the original call.

Comment (pirapira) [reply src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_following_too_big_transferFiller.json:33]: I'll add these to the spreadsheet, and write a comment here again.

Comment (pirapira) [reply src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_following_too_big_transferFiller.json:33]: Case 1. https://docs.google.com/spreadsheets/d/1xat7UI8GtB4ZGVdlK5_XQSHJZaMThi4SrlcL8XMZb5Q/edit#gid=1605675414&range=C36

Comment (pirapira) [reply src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_following_too_big_transferFiller.json:33]: I didn't understand the second case you described.

Comment (winsvega) [reply src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_following_too_big_transferFiller.json:33]: TRANSACTION -> CALL -> (CALL -> CALL RETURN | OOG) | check return data

-> mean call
| finish call and go one level up

**Pull request merged**